documentation/docs/getting-started/using-extensions.md

```diff
@@ -300,9 +300,134 @@ You can enable or disable installed extensions based on your workflow needs.
 </Tabs>
 
 :::tip
-Goose can automatically detect and suggest the right extensions for your tasks as you chat. Learn more about this smart feature in the [Smart Extension](/docs/guides/smart-extension-recommendation) guide.
+Goose can automatically detect and suggest the right extensions for your tasks as you chat. To learn more, see [Automatically Enabled Extensions](#automatically-enabled-extensions).
```

Comment:
perhaps this tip is redundant now that the section for this is immediately underneath the tip. I think we can remove this